### PR TITLE
Add PyTorch Hub configuration file

### PR DESCRIPTION
**Pull request by johnsutor**
This PR aims to add support for PyTorch's hub loading feature. This would make model loading as simple as specifying this git repo to download the CLIP model along with its pre-trained weights. When working with `torch.hub.load`, the example would change from this

```py
import torch
import clip
from PIL import Image

device = "cuda" if torch.cuda.is_available() else "cpu"
model, preprocess = clip.load("ViT-B/32", device=device)

image = preprocess(Image.open("CLIP.png")).unsqueeze(0).to(device)
text = clip.tokenize(["a diagram", "a dog", "a cat"]).to(device)

with torch.no_grad():
    image_features = model.encode_image(image)
    text_features = model.encode_text(text)
    
    logits_per_image, logits_per_text = model(image, text)
    probs = logits_per_image.softmax(dim=-1).cpu().numpy()

print("Label probs:", probs)  # prints: [[0.9927937  0.00421068 0.00299572]]
```

to this 

```py
import torch
from PIL import Image

device = "cuda" if torch.cuda.is_available() else "cpu"
model, preprocess = torch.hub.load("openai/CLIP", "ViT_B_32", device=device)
tokenize = torch.hub.load("openai/CLIP", "tokenize")

image = preprocess(Image.open("CLIP.png")).unsqueeze(0).to(device)
text = tokenize(["a diagram", "a dog", "a cat"]).to(device)

with torch.no_grad():
    image_features = model.encode_image(image)
    text_features = model.encode_text(text)
    
    logits_per_image, logits_per_text = model(image, text)
    probs = logits_per_image.softmax(dim=-1).cpu().numpy()

print("Label probs:", probs)  # prints: [[0.9927937  0.00421068 0.00299572]]

### CODE REVIEW

**Comment by jongwook** [comment]
It works! Thanks for the contribution :)
